extensions/amp-sona/0.1/test/validator-amp-sona.html

```diff
@@ -0,0 +1,42 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Can you update this value to 2020? The value should match the year the new file is committed.

Reply:
👍